Documentation/release-notes/4054.md

```diff
@@ -0,0 +1,24 @@
+## Support for Binding Kotlin Libraries
+
+This release adds better support for creating bindings for Kotlin libraries, including:
+
+* Hiding Kotlin `internal` types and members
+* Using Kotlin provided metadata to properly name method parameters
+* Hiding Kotlin generated methods like `foo-impl` that aren't intended for public use
+* Restoring Kotlin-mangled method names to their original name (ie: `foo-V5j3Lk8` -> `foo`)
+* Adding support for Kotlin's native unsigned types (UInt, ULong, UShort, UByte)
```

Comment:
I think these bullet points are excellent, by the way. Interesting and clear info.